internal/config/config.proto

```diff
@@ -103,7 +103,20 @@ message MetricsOptions {
   // How often to export the metrics to Google Cloud.
   google.protobuf.Duration export_interval = 2;
 }
+
+// Options to configure how the logs are handled by the deployer.
+message LoggingOptions {
+  // Minimum level for a log entry to be exported to Google Cloud Logging.
+  //
+  // Acceptable values are: "DEBUG", "INFO", "WARN", "ERROR". Note that these
+  // are the acceptable values by slog [1].
+  //
+  // [1] https://pkg.go.dev/log/slog#Level
+  string min_export_level = 1;
```

Comment:
I like the way you have this! But, if you feel like over-engineering, I think log levels can actually be any integer: https://pkg.go.dev/golang.org/x/exp/slog#Level. So if a user logs with a custom log level below DEBUG, they won't have a way to enable those logs. I don't think it's worth changing though :)

Reply:
Great point. I think very few people will set the log level in the config. We can revisit this later if people really need it.